docs/guides/user-guide/available-environments.md

```diff
@@ -15,13 +15,15 @@ Depending on the environment, StackBlitz IDE includes slightly different feature
 
 | Feature | EngineBlock | WebContainers |
 | --- | --- | --- |
-| Native Node.js environment | <SupportIcon value="no" label="Not available" /> | <SupportIcon value="yes" label="Available" /> |
 | Supported frameworks | <SupportIcon value="star-half" label="" /> Front-end | <SupportIcon value="star" label="" /> Front-end & back-end |
-| Editor | <SupportIcon value="yes" label="Available" /> | <SupportIcon value="yes" label="Available" /> |
-| Preview | <SupportIcon value="yes" label="Available" /> | <SupportIcon value="yes" label="Available" /> |
-| Shareable Preview URL | <YepNope value="yes" label="Supported" /> | <SupportIcon value="no" label="Not supported" /> |
-| Console | <SupportIcon value="yes" label="Available" /> | <SupportIcon value="no" label="Not available" /> |
-| Terminal | <SupportIcon value="no" label="Not available" /> | <SupportIcon value="yes" label="Available" /> |
+| Supported package managers | <SupportIcon value="star-half" label="" /> Turbo v1 | <SupportIcon value="star" label="" /> Turbo v2, pnpm, yarn v1 |
+| Full Node.js environment | <SupportIcon value="no" label="Not available" /> | <SupportIcon value="yes" label="Available" /> |
+| Classic editor | <SupportIcon value="yes" label="Available" /> | <SupportIcon value="yes" label="Available" /> |
+| [Codeflow](/codeflow/working-in-codeflow-ide) (beta) | <SupportIcon value="no" label="Not available" /> | <SupportIcon value="yes" label="Available" /> |
+| [Web Publisher](/codeflow/content-updates-with-web-publisher) (beta) | <SupportIcon value="no" label="Not available" /> | <SupportIcon value="yes" label="Available" /> |
+| Shareable preview URL | <SupportIcon value="yes" label="Available" /> | <SupportIcon value="no" label="Not available" /> |
+| JavaScript console | <SupportIcon value="yes" label="Available" /> | <SupportIcon value="no" label="Not available" /> |
```

Comment:
You're right - if we call it "DevTools console", this may confuse people with the actual browser's console. I like the simple "Console" but maybe "JS Console" works too?

Hmmmm... @sulco do you have strong opinions, preferences, or anecdotal evidence to share?

Reply:
No strong opinions. The only anegdote I can think of is that it's called devTools in our [sdk](https://developer.stackblitz.com/guides/integration/embedding#embed-url-options) so it's worth unifying that one way or the other.

Reply:
Ah, actually I have one more 😅 It's not out of the realm of possibility that it will in fact be extended with other devtools-y tools. We've just recently [had this demoed and briefly discussed](https://stackblitz.slack.com/archives/C01MYU0R92T/p1663529220565099), so one concern is that we might have to change it back in several months.

Reply:
> The only anegdote I can think of is that it's called devTools in our [sdk](https://developer.stackblitz.com/guides/integration/embedding#embed-url-options) so it's worth unifying that one way or the other.

I purposefully moved away from using "DevTools" in our SDK, but I can't change the names of the options themselves because that would be a breaking API change.

Reply:
I changed the name of the Console and Terminal headers back to just “Console” and “Terminal”, but linked those words to the corresponding sections in the “IDE: what's on your screen” page.